internal/pgbackrest/config_test.go

```diff
@@ -43,6 +43,10 @@ func TestCreatePGBackRestConfigMapIntent(t *testing.T) {
 	cluster.Spec.Port = initialize.Int32(2345)
 	cluster.Spec.PostgresVersion = 12
 
+	cluster.Labels = map[string]string{
+		naming.LabelVersion: "2.3.0",
```

Comment:
will we need to update this value (in other tests too) after every release?

Reply:
No, since after this version, we want labels to be added, starting with 2.3.0.